tests/test.h

```diff
@@ -149,4 +149,15 @@ void same_param_value(char* dev_name, uint64_t uid, char* param_name, param_type
  */
 void check_param_range(char* dev_name, uint64_t uid, char* param_name, param_type_t param_type, param_val_t expected_low, param_val_t expected_high);
 
+/**
+*Checks the latency between an action and the parameter it affects being changed
+ * Arguments:
+ *    dev_name: The name of the device whose parameters are being checked
+ *    uid: uniquie identifier of device
+ *    param_name: The name of the parameter being compared
+ *    lower_bound_latency: The expected lowerbound latency 
+ *    upper_bound_latency: The expected upperbound latency
+ *    start_time: The time right before the button is pressed
+*/
+void check_latency(char* dev_name, uint64_t uid, char* param_name, int32_t lower_bound_latency, int32_t upper_bound_latency, int32_t start_time);
```

Comment:
I think adding this function is a good idea! From what I can tell though, this function will be used only for devices with some kind of `TIMESTAMP` parameter like the `TimeTestDevice` has, right?

We should add a comment explaining that the parameter has to be specifically something that holds a timestamp to be used as the "end time"

Reply:
Good idea, I'll add in a comment on that